src/theme/components/header/header.ts

```diff
@@ -11,7 +12,7 @@ export const headerFocus = {
   outlineColor: `${headerDarkBlue} !important`,
   outlineOffset: "0 !important",
   outlineStyle: "solid !important",
-  outlineWidth: "0.1875em !important",
+  outlineWidth: "0.25em !important",
 };
 
 const Header = {
```

Comment:
The `fontFamily` attribute below needs to be updated.  `’system ui’` should be changed to `’system-ui’`.